.docker/jats-ingester/airflow.cfg

```diff
@@ -1,13 +1,13 @@
 [libero]
-# url to override boto's default of aws s3. If blank will use boto default.
-boto_endpoint_url =
```

Comment:
was this necessary before? Only if so, would like for it to remain here for backward compatibility until we have removed the dependency

Reply:
This was replaced by using the airflow `remote_logs` connection but I must have forgotten to  remove this setting. It wasn't present in the dev config file.

Just thinking about it, we could just make this an environment variable in docker compose?

Reply:
ok, this can go